app/ui/src/app/platform/types/status-code-decoder/status-code-decoder.service.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+import { LeveledMessage } from '@syndesis/ui/platform';
+
+@Injectable()
+export abstract class StatusCodeDecoderService {
+  abstract getMessageString(message: LeveledMessage, context?: any);

```

Comment:
You've typed `context` as `StringMap`somewhere above this file. I'd use the same type annotation here, if possible, for the sake of consistency.

Reply:
Good catch!  Updated that...